src/usr/local/buildpack/tools/elixir.sh

```diff
@@ -5,12 +5,17 @@ set -e
 require_root
 check_command erl
 
+if [[ -d "/usr/local/${TOOL_NAME}/${TOOL_VERSION}" ]]; then
+  echo "Skipping, already installed"
+  exit 0
+fi
```

Comment:
Is this specific to elixir? Seems like a generic rule. Easier copy/paste, less chance of human error?

Reply:
not yet really happy with this, as for some tools i like to support multiple versions and latest install call should override env again, so executables in path always points to latest installed version.

see python or php for a more generic version. will extract some more helper functions later to make new tools even more easy to implement.